libraries/AP_HAL_ESP32/boards/esp32s3devkit.h

```diff
@@ -14,166 +14,138 @@
  */
 #pragma once
 
+#define HAL_ESP32_BOARD_NAME "esp32s3devkit"
 #define CONFIG_HAL_BOARD_SUBTYPE HAL_BOARD_SUBTYPE_ESP32_S3DEVKIT
 // make sensor selection clearer
-#define PROBE_IMU_I2C(driver, bus, addr, args ...) ADD_BACKEND(AP_InertialSensor_ ## driver::probe(*this,GET_I2C_DEVICE(bus, addr),##args))
-#define PROBE_IMU_SPI(driver, devname, args ...) ADD_BACKEND(AP_InertialSensor_ ## driver::probe(*this,hal.spi->get_device(devname),##args))
-#define PROBE_IMU_SPI2(driver, devname1, devname2, args ...) ADD_BACKEND(AP_InertialSensor_ ## driver::probe(*this,hal.spi->get_device(devname1),hal.spi->get_device(devname2),##args))
 
-#define PROBE_BARO_I2C(driver, bus, addr, args ...) ADD_BACKEND(AP_Baro_ ## driver::probe(*this,std::move(GET_I2C_DEVICE(bus, addr)),##args))
-#define PROBE_BARO_SPI(driver, devname, args ...) ADD_BACKEND(AP_Baro_ ## driver::probe(*this,std::move(hal.spi->get_device(devname)),##args))
+#define AP_RCPROTOCOL_ENABLED 1
+
+#define AP_RCPROTOCOL_IBUS_ENABLED 1
```

Comment:
This particular board was created by @nkruzan , and he presumably has hardware that's wired a particular way. 
I dont believe its pinout is documented in any sort of readme at this stage ( unfortunately), so its a bit of guesswork wether he's going to be concerned by having sbus and ppmsum "turned off" on his hardware.  
 I suggest that documenting a public pinout for your board/hardware and using your own specific build-target that matches it is better than re-using this one in this case.  eg README.esp32s3arg and using --board=esp32s3arg   [ or whatever you call it ].
It would be great to have a 's3' target that has its pinout/s documented better.  
conversely, we could ask @nkruzan  how he feels about you re-using his board-type, and he might be fine with it. :-)

Reply:
I think it would be better to enhance existing boards (adding pin and sensor description), than proliferating new one for each developer.

If nobody objects to it.